GithubEmojiCompletions.py

```diff
@@ -55,5 +83,18 @@ def is_valid_scope(view, point):
 
 
 def plugin_loaded():
-    global settings
+    global settings, emojis_alias, emojis_emoji, emojis, commit_emojis_alias, commit_emojis_emoji
     settings = sublime.load_settings("GithubEmoji.sublime-settings")
+    emojis = sublime.decode_value(sublime.load_resource('Packages/GithubEmoji/emoji.json'))
+    emojis_alias = list(map(lambda emo: [":{}: {}\t{}".format(*emo), emo[0] + ':'], emojis))
+    emojis_emoji = list(map(lambda emo: [":{}: {}\t{}".format(*emo), emo[1]], emojis))
+
+    commit_emojis = sublime.decode_value(
+                        sublime.load_resource('Packages/GithubEmoji/commit-emoji.json'))
+    commit_emojis_alias = []
+    commit_emojis_emoji = []
+    for emo in commit_emojis:
+        commit_emojis_alias.append([":{}: {}\t{}".format(*emo), ":{}:".format(emo[0])])
+        commit_emojis_alias.append([":{}: {}\t{}".format(*reversed(emo)), ":{}:".format(emo[0])])
```

Comment:
You can omit the colon here: `commit_emojis_alias.append(["{} {}\t{}".format(*reversed(emo)), ":{}:".format(emo[0])])`